WebContent/WEB-INF/jsp/controlPanel.jsp

```diff
@@ -143,7 +143,7 @@
 									   <span class="submenuText"><input type="radio" name="size" value="2" > Service (768px)</span><br />
 									   <span class="submenuText"><input type="radio" name="size" value="6" > Large (1200px)</span><br />
 									   <span class="submenuText"><input type="radio" name="size" value="7" > Huge (1600px)</span><br />
-									   <span class="submenuText"><input type="radio" name="size" value="v" > Videos mp4 (720p) </span><br />
+									   <span class="submenuText"><input type="radio" name="size" value="v" > MP3/MP4 </span><br />
```

Comment:
No. It's not related to the filestore change but just text correction since I see it outdated per the function provided. I can make it in another commit.